devDocs/displayModel.md

```diff
@@ -0,0 +1,18 @@
+# Display model
+
+The display model (part of NVDA Helper), is a legacy mechanism for scraping
+information during the rendering of GUI applications.
+
+For information on working in this area see the `NVDAHelper/readme.md` file.
+
+## Testing GDI applications
+To test:
+- Use `NVDA+numPad 7` to enter screen review mode.
+- Use the number pad to read lines (7, 8, 9), read words, (4, 5, 6), or characters (1, 2, 3).
+
+Note:
+- For color reporting enable the option (via Document formatting panel) _"report color"_.

```

Comment:
Would it be worth mentioning here that for quick checking of color NVDA+Shift+F can be used?

Reply:
Yes, as long as we are confident it reports the color from the display model? I'll have to test to be sure.

Reply:
I have confirmed this.